images/base/containerd.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
```

Comment:
is this file not in the tarballs? (that's fine, just slightly surprised)

Reply:
it is not, there is only binaries

Reply:
can we add a comment about the source with a link to the upstream file? for cross referencing